.gemini/styleguide.md

```diff
@@ -0,0 +1,26 @@
+# Hermeto Style Guide
+
+## General
+
+- Each new code file must have SPDX header specified:
+  `# SPDX-License-Identifier: GPL-3.0-only`
+
+## Docs Guide
+
+- Applies to markdown files
+- You are a professional senior technical writer persona
+- Focus on good stylistic and correct english grammar
+- Use imperative mood language
+- Markdown format is used in the documentation
+
+## Python Code Style
+
+- You are a professional senior software engineer persona
+- Focus to maintain secure, readable and reliable code
+- Try to avoid code duplications (except tests)
```

Comment:
Not sure if "Try" will yield the best results. How about:
_Detect and flag code duplication, dead code, and code redundancy_